File: app/src/main/java/com/neome/api/meta/base/dto/StudioEntAutomationStepCopyField.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.MetaIdGrid;
import com.neome.api.meta.base.Types.MetaIdPipelineParam;
import com.neome.api.meta.base.Types.MetaIdVar;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioEntAutomationStepCopyField extends StudioEntAutomationStep
{
  @Nullable
  public MetaIdVar mappingVarId;

  @Nullable
  public StudioValueVarIdCondition sourceGridFilterVarId;

  @Nullable
  public MetaIdGrid sourceGridId;

  @Nullable
  public MetaIdPipelineParam sourcePipelineVarId;

  @Nullable
  public MetaIdPipelineParam targetPipelineVarId;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioEntAutomationStepCopyField.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdGrid
import com.neome.api.meta.base.Types.MetaIdPipelineParam
import com.neome.api.meta.base.Types.MetaIdVar
import com.neome.api.meta.base.dto.StudioEntAutomationStep
import com.neome.api.meta.base.dto.StudioValueVarIdCondition

interface StudioEntAutomationStepCopyField : StudioEntAutomationStep
{
  val mappingVarId: MetaIdVar?
  val sourceGridFilterVarId: StudioValueVarIdCondition?
  val sourceGridId: MetaIdGrid?
  val sourcePipelineVarId: MetaIdPipelineParam?
  val targetPipelineVarId: MetaIdPipelineParam?
}